maven-plugin/src/main/maven/plugin.xml

```diff
@@ -92,8 +92,8 @@
         <generateServer implementation="boolean" default-value="true">${akka-grpc.generate-server}</generateServer>
         <generatorSettings implementation="java.util.List" default-value=""/>
         <extraGenerators implementation="java.util.List" default-value=""/>
-        <protoPaths default-value="src/main/proto,src/main/protobuf">${akka-grpc.protoPaths}</protoPaths>
-        <outputDirectory default-value="target/generated-sources">${akka-grpc.outputDirectory}</outputDirectory>
+        <protoPaths default-value="${project.build.sourceDirectory}/../proto,${project.build.sourceDirectory}/../protobuf">${akka-grpc.protoPaths}</protoPaths>
```

Comment:
I may be wrong but looking at a few other plugins they seem to rather do what would be `${basedir}/src/main/proto`, would that also break with multi-modules?

Reply:
you are right, aligned the definition to follow the reference pom:
https://maven.apache.org/pom.html